addon/components/horizontal-bar-chart.js

```diff
@@ -415,7 +419,7 @@ const HorizontalBarChartComponent = ChartComponent.extend(FloatingTooltipMixin,
     const maxValueLabelWidth = this._maxWidthOfElements(valueLabelElements);
     const maxGroupLabelWidth = this._maxWidthOfElements(groupLabelElements);
 
-    const maxLabelWidth = this.get('maxLabelWidth');
+    const maxLabelWidth = this.get('maxLabelWidth') || this.get('outerWidth');
```

Comment:
Don't use the or operator to defaulting ever. You can replace this with `this.getWithDefault(...)`